modules/apps/frontend-js/frontend-js-tooltip-support-web/src/main/resources/META-INF/resources/index.js

```diff
@@ -104,6 +104,34 @@ const TooltipProvider = () => {
 		return dispose;
 	}, [delay, state]);
 
+	const getContent = element => {
+		if (element) {
+			const title = element.getAttribute('title');
+
+			if (title) {
+				element.setAttribute('data-title', title);
```

Comment:
This will collapse 2 different starting situations into just 1 making it impossible to distinguish later when restoring the title is needed.

```
<div title="foo">...</div>
<div data-title="bar">...</div>
```

With this change when we restore we'll end up with:

```
<div title="foo">...</div>
<div title="bar">...</div>
```

Which is certainly different than the starting point.

We should use a different data attribute to store `title` so we know what to restore and we leave `data-title` alone. Something like `data-lfr-saved-tooltip-title` or `data-restore-title` or anything you want to come up with.

Reply:
Thanks. This explains why we had `data-restore-title` in the previous version.
I'll change that